src/components/Character/renderUtils.jsx

```diff
@@ -8,7 +8,7 @@ import {
 import { s } from "./";
 
 
-function renderButton(name, direction){
+function renderButton(name, direction, newIndex){
```

Comment:
If we're going to just give it the `index` then there's no point in having the `direction` parameter and we might as well just pass a `label` or either "next" or "previous" - let's either move the index calculations into here and use direction, or remove the direction and add label.

Reply:
Dooone